susApp/www/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
+
+    <!-- To change the colour of searchbox and statusbar in Gocle Chrome browser -->
+    <meta name="theme-color" content="#348fce">
+    <title></title>
+
+    <!-- CSS Links -->
+    <link href="css/ionic.app.css" rel="stylesheet">
+    <link href="lib/ionic/css/ionic.css" rel="stylesheet">
+
+    <!-- Required js libraries -->
+    <script src="lib/ionic/js/ionic.bundle.js"></script>
+    <script src="cordova.js"></script>
+    <script src="js/ng-cordova.js"></script>
+    <script src="js/ngStorage.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+
+    <!-- App js files -->
+    <script src="js/app.js"></script>
+    <script src="js/Postitem.js"></script>
+    <script src="js/Login.js"></script>
+    <script src="js/Register.js"></script>
+    <script src="js/Submitform.js"></script>
+    <script src="js/Userinput.js"></script>
+    <script src="js/Validation.js"></script>
+    <script src="js/RequestReset.js"></script>
+    <script src="js/ResetPass.js"></script>
+    <script src="js/Message.js"></script>
+    <script src="js/Services.js"></script>
+    <script src="js/test.js"></script>
+    <script src="js/Getitems.js"></script>
+    <script src="js/Viewitem.js"></script>
+    <script src="js/Myitems.js"></script>
+    <script src="js/DrawerLinks.js"></script>
+    <script src="js/Useraccount.js"></script>
+
+    <script src="lib/angular-cache/dist/angular-cache.js"></script>
+    <script src="js/MainpageServices.js"></script>
+
+
```

Comment:
Remove extra space. This applies to various files where there are two concurrent spaces that are not needed